packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -536,6 +542,9 @@ class ScrollableBaseNative extends Component {
 			childContainerRef.addEventListener('mouseover', this.onMouseOver, {capture: true});
 			childContainerRef.addEventListener('mousemove', this.onMouseMove, {capture: true});
 			childContainerRef.addEventListener('focusin', this.onFocus);
+			childContainerRef.addEventListener('webOSVoice', this.onVoice);
+			// FIXME In case of VirtualList, addEventListener target is not matched childContainerRef. So, scroll intent is added dynamically.
+			childContainerRef.setAttribute('data-webos-voice-intent', 'Scroll');
```

Comment:
It might be needed to add condition (e.g> platform.tv) for any TV specific features. For other devices, we don't have to handle them.

```
import {platform} from '@enact/webos/platform';

if (platform.tv) {
	childContainerRef.addEventListener('webOSVoice', this.onVoice);
	FIXME In case of VirtualList, addEventListener target is not matched childContainerRef. So, scroll intent is added dynamically.
	childContainerRef.setAttribute('data-webos-voice-intent', 'Scroll');
}

Reply:
I prefer setting attributes in `componentDidMount` and/or `componentDidUpdate' if possible, but it looks not bad here.

Reply:
I did add condition(platform.tv) base on your guide. Thanks